### PR TITLE
Add instructions for running on android via Termux

### PR DESCRIPTION
**Pull request by vbajs**
This PR adds instructions to run the script on Termux.
A final thing from my (now deprecated) fork

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,4 +54,14 @@ type = 0
 url = https://twitch.tv/username
 # can only be online, idle or dnd to make the rich presence work
 status = online
-```
\ No newline at end of file
+```
+
+# Termux instructions
+1. Download Termux from F-Droid (you can also download from the play store but that's outdated) 
+Optional: download hacker keyboard from playstore so that moving is easier, though, vim has touch support so only use hacker keyboard if you are going to use nano instead
```
Comment (kairusds) [inline README.md:61]: you could also run `mkdir $HOME/.termux/ ;echo "extra-keys = [['ESC','/','-','HOME','UP','END','PGUP'],['TAB','CTRL','ALT','LEFT','DOWN','RIGHT','PGDN']]" >> $HOME/.termux/termux.properties && termux-reload-settings && sleep 1 && logout` if you want to add arrow keys

Comment (vbajs) [reply README.md:61]: As I mentioned, I never needed arrow keys due to vim having native touch support.
Though, this is worth mentioning

**Comment by kairusds** [comment]
tysm

File: README.md (hunk 1 above)
Comment (kairusds) [reply README.md:61]: oh alright

**Pull request merged**